extensions/search/README.md

```diff
@@ -0,0 +1,26 @@
+# Search Extension Specification
+
+**Extension [Maturity Classification](../../../extensions/README.md#extension-maturity): Proposal**
+
+This extension is intended to augment the core ItemCollection object when the ItemCollection is the result of a 
+search, for example, from calling the `/stac/search` API endpoint.
+
+- [Example](examples/example.json)
+- [JSON Schema](json-schema/schema.json)
+
+### ItemCollection fields
+
+| Element      | Type          | Description                                                  |
+| ------------ | ------------- | ------------------------------------------------------------ |
+| `search:metadata` | SearchMetadata Object | **REQUIRED.** The search-related metadata for the [ItemCollection](../itemcollection-spec/README.md). |
+
+### SearchMetadata Object
+
+| Element      | Type          | Description                                                  |
+| ------------ | ------------- | ------------------------------------------------------------ |
+| next         | string\|null  | **REQUIRED.** The value to set for the `next` query parameter in order to get the next page of results. |
```

Comment:
It is not documented what `null` is about.

Reply:
Also, should we also allow integers here?

Reply:
I dislike multi-typed attributes.  In practice, this is representing a query string parameter, so it's a string, even if it's a string of digits like a `page`/`offset` value would be.

Reply:
True. It's the response...